### ISSUE TITLE
Just Tried to Audit and Received an Error That Prevents Me From Loading the Page

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#332]
This seems pretty serious. Is anyone able to audit anything on Project Sidewalk right now. Even when I do a ctrl-refresh, I get this error:

![image](https://cloud.githubusercontent.com/assets/1621749/19198889/5bc39a38-8c8f-11e6-8d25-39e917bcc292.png)

And none of the buttons seem to work (e.g., I can't click on the Feedback button).

![image](https://cloud.githubusercontent.com/assets/1621749/19198939/9a681ef8-8c8f-11e6-8181-46221436c43c.png)

**Comment by manaswis** [comment]
Yes, its not working! :( 

Soheil and I are looking into it now.

**Comment by jonfroehlich** [comment]
Thanks for investigating. 

This brings up an additional important point that we should probably have one person try to login and complete an audit at least once every day or every other day to ensure that things are still working.

**Comment by manaswis** [comment]
Something really wierd..its erratic. It started working for me again. We are trying to figure out what's happening.

**Comment by manaswis** [comment]
Yeah, I agree. I usually check the admin every day. I'll do the same for audit.

**Comment by Soben713** [comment]
Something strange is happening, the `neighbourhoodModel` could be two totally different objects. And that's what causing the problem. This is when it is not working:
![image](https://cloud.githubusercontent.com/assets/1576529/19199600/c925ae74-8c92-11e6-8eb9-ec5d7872f03d.png)
And this is when it is:

**Comment by manaswis** [comment]
The problem is resolved for now. But we are still testing it.

Couple of things that happened. 

1. The errors that are reported in the comments above was generated by a version of SVLabel.js file that was outdated. The newer version isn't giving any issues. Our hunch is that the older version was getting loaded from the cache. We are investigating that issue on our local servers and on the remote server.

2. There is a load balancing in place for 9000 and 9001 ports. So the pages that are loaded come either 9000 server instance or 9001. Since 9001 had the latest version, that might be a cause (we aren't sure). So for now, we have shut down the server instance running on 9001. 

The errors aren't coming now.

**Comment by manaswis** [comment]
TLDR;
Load balancing between the two server instances, running different versions of the site, was creating the issue.

Details:
After more testing, here are the conclusions:

What we believed as the older version of SVLabel.js (see the previous comment) is actually the newer version, and it isn't creating any issues on our local servers and on the remote server (running on a different port for testing). So my strong guess is that due to _load balancing_, the page request (e.g. clicking on audit button) would have come from the 9000 version (which is running an old version of the app) and the request was served by the 9001 version. Due to the version mismatch between the two servers, the errors were generated.

Now, since the 9001 version is shutdown, all requests are served by the 9000 version. And on our local servers, the latest version. Both aren't giving any issues.

We can close this issue.

**Comment by Soben713** [comment]
I just had the same experience on my local server today, so I don't think it is really related to server's load balancing or at least that's not the only possible reason. I recorded a video of what happened and [here](https://www.youtube.com/watch?v=nED4IsDHyCo) is the link to it.

Interestingly in Firefox, there is no problem when I keep my developer tools open when reloading the page, but it occurs if it is closed. The reason is that the cache is disabled when developer tools window is opened. Disabling the cache on Safari seems to fix the problem too. 

Basically, a wrong version of SVLabel.js file is served every time this problem occurs and there are a lot of differences between the two versions. Here is a screenshot of the diff:

![image](https://cloud.githubusercontent.com/assets/1576529/19445753/a797705c-9463-11e6-9224-256db5c58300.png)

I am going to open this issue again until we fully understand it and make sure that it does not happen again.
